test/dlc_tests/container_tests/bin/testTensorflow2Standalone

```diff
@@ -26,17 +27,22 @@ def check_tensorflow():
         sess = tf.compat.v1.Session(config=tf.compat.v1.ConfigProto(log_device_placement=False))
 
         with sess:
+            tf_version = tf.__version__
+            version_check = r"2.[0-3]+"
+            device_tag = ''
+            if(re.search(version_check, tf_version)):
```

Comment:
Use `re.match` instead of `re.search`, so that the regex pattern doesn't match substrings in version numbers such as `"2.2.3"`.
```suggestion
            if(re.match(version_check, tf_version)):
```